datafusion/datasource-parquet/src/sort.rs

```diff
@@ -153,7 +166,10 @@ mod tests {
         let selection =
             RowSelection::from(vec![RowSelector::select(50), RowSelector::skip(250)]);
 
-        let reversed = reverse_row_selection(&selection, &metadata).unwrap();
+        let row_groups_to_scan = vec![0, 1, 2];
```

Comment:
Isn't `row_groups_to_scan` derived from the `RowSelection`? So in this case it should be `vec![0]`?

Reply:
Ok, i got it now @adriangb , i will try to address it, thanks!

Reply:
Addressed in latest PR.